R/queue.R

```diff
@@ -36,14 +36,17 @@ Queue <- R6::R6Class("Queue", # nolint
     #' @param parameters Parameters to run the report with.
     #' @param location Location of the outpack repository from which to pull
     #'   dependencies and push the produced packet.
-    submit = function(url, branch, ref, reportname, parameters, location) {
+    #' @param ssh_key private SSH key as a character vector (separated by newline characters)
```

Comment:
```suggestion
    #' @param ssh_key private SSH key as a character scalar
```

I don't think the newline separation is important or relevant here (in fact the newlines are probably optional, you can join the lines and it would work fine). The fact that it is a length 1 vector is though.